src/libs/Permissions.ts

```diff
@@ -27,6 +27,7 @@ function canUseTrackExpense(betas: OnyxEntry<Beta[]>): boolean {
 }
 
 function canUseP2PDistanceRequests(betas: OnyxEntry<Beta[]>): boolean {
+    return true;
```

Comment:
Fine for testing, but let's not commit this.

Reply:
fixed